scripts/runTestsOnTravis.sh

```diff
@@ -3,17 +3,12 @@
 set -e
 
 if [ "$1" = "compile" ]; then
-
-    # We will compile and run some vet, spelling and some other checks.
-
-    go install honnef.co/go/tools/cmd/staticcheck@latest;
-    go install github.com/client9/misspell/cmd/misspell@latest;
-    GO_LIST=$(go list ./...);
+    # First check that NATS builds.
     go build;
-    $(exit $(go fmt $GO_LIST | wc -l));
-    go vet $GO_LIST;
-    find . -type f -name "*.go" | xargs misspell -error -locale US;
-    staticcheck -tags=js_chaos_tests $GO_LIST
+
+    # Now run the linters.
+    curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.50.1
```

Comment:
we could do `go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest` but this binary is about 30mb so prefer installing via the script too and just getting the release

Reply:
Yeah, I did consider using `go install` but the `golangci-lint` documentation advises against it, mostly because it's considerably slower than just fetching a binary.

Reply:
How much time does it add to the travis runs?

Reply:
About 20-30 seconds.

Reply:
ok will check the timing differences on your current run. Thanks.